gstd/src/async_runtime/futures.rs

```diff
@@ -68,6 +68,7 @@ where
 
     if Pin::new(&mut task.future).poll(&mut cx).is_ready() {
         super::futures().remove(&msg_id);
+        super::locks().remove_message_entry(msg_id);
```

Comment:
One more place to remove lock
https://github.com/gear-tech/gear/blob/7c0ae01484e77bf4d5f5d7dcd830829b68011263/gstd/src/async_runtime/mod.rs#L58-L60

Reply:
added